assets/src/application/ui/display/ItemCount/index.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Badge } from 'antd';
 import { BadgeProps } from 'antd/lib/badge';
```

Comment:
can we just manually copy whatever these types are directly into our type declarations so that we can drop AntD as a dependency? Or is this even needed? Can't imagine why would still be using AntD props...

Reply:
Updated types.